interface/main/onotes/office_comments.php

```diff
@@ -19,18 +19,22 @@
  */
 
 include_once("../../globals.php");
-include_once("$srcdir/onotes.inc");
 include_once("$srcdir/formatting.inc.php");
 
 //display all of the notes for the day, as well as others that are active from previous dates, up to a certain number, $N
 $N = 10;
+
+$oNoteService = new \services\ONoteService();
 ?>
 
 <html>
 <head>
 
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
```

Comment:
just the css file:
https://github.com/morteza/bootstrap-rtl/blob/master/README.md#bootstrap-3-rtl-theme